Apps/Sandcastle/gallery/Time Dynamic Wheels.html

```diff
@@ -0,0 +1,129 @@
+viewer.clock.stopTime = stop.clone();
+viewer.clock.currentTime = start.clone();
+viewer.clock.clockRange = Cesium.ClockRange.LOOP_STOP;
+viewer.timeline.zoomTo(start, stop);
+
+// Create a path for our vehicle by lerping between two positions.
+var position = new Cesium.SampledPositionProperty();
+var startPosition = new Cesium.Cartesian3(-2379556.799372864, -4665528.205030263, 3628013.106599678);
+var endPosition = new Cesium.Cartesian3(-2379603.7074103747, -4665623.48990283, 3627860.82704567);
+// A velocity vector property will give us the entity's speed and direction at any given time.
+var velocityVectorProperty = new Cesium.VelocityVectorProperty(position, false);
+var velocityVector = new Cesium.Cartesian3();
+// Store the wheel's rotation over time in a SampledProperty.
+var wheelAngleProperty = new Cesium.SampledProperty(Number);
+var wheelAngle = 0;
+
+var numberOfSamples = 100;
+for (var i = 0; i <= numberOfSamples; ++i) {
+    var factor = (i / numberOfSamples);
+    var time = Cesium.JulianDate.addSeconds(start, factor * totalSeconds, new Cesium.JulianDate());
+
+    // Lerp using a non-linear factor so that the vehicle accelerates.
+    var locationFactor = Math.pow(factor, 2);
+    var location = Cesium.Cartesian3.lerp(startPosition, endPosition, locationFactor, new Cesium.Cartesian3());
+    position.addSample(time, location);
+    // Rotate the wheels based on how fast the vehicle is moving at each timestep.
+    velocityVectorProperty.getValue(time, velocityVector);
+    var metersPerSecond = Cesium.Cartesian3.magnitude(velocityVector);
+    var wheelRadius = 0.52;//in meters.
+    var circumference = Math.PI * wheelRadius * 2;
+    var rotationsPerSecond = metersPerSecond / circumference;
+
+    wheelAngle += ((Math.PI * 2 * totalSeconds) / numberOfSamples) * rotationsPerSecond;
+    wheelAngleProperty.addSample(time, wheelAngle);
+}
+
+function updateSpeedLabel(time, result) {
+    velocityVectorProperty.getValue(time, velocityVector);
+    var metersPerSecond = Cesium.Cartesian3.magnitude(velocityVector);
+    var kmPerHour = Math.round(metersPerSecond * 3.6);
+
+    return kmPerHour + ' km/hr';
+}
+
+var rotationProperty = new Cesium.CallbackProperty(function(time, result) {
+    return Cesium.Quaternion.fromAxisAngle(Cesium.Cartesian3.UNIT_X, wheelAngleProperty.getValue(time), result);
+}, false);
+
+var wheelTransformation = new Cesium.NodeTransformationProperty({
+    rotation: rotationProperty
+});
+
+var nodeTransformations = {
+    'Wheels': wheelTransformation,
+    'Wheels_mid': wheelTransformation,
+    'Wheels_rear': wheelTransformation
+};
+
+// Add our vehicle model.
+var vehicleEntity = viewer.entities.add({
+    position : position,
+    orientation : new Cesium.VelocityOrientationProperty(position), // Automatically set the vehicle's orientation to the direction it's facing.
+    model : {
+        uri : '../../../../Apps/SampleData/models/GroundVehicle/GroundVehicle.glb',
+        minimumPixelSize : 128,
```

Comment:
This and maximumScale just add news to the demo, since we automatically track the entity, there's no reason to set these, users will see it no problem.